### DISCUSSION TITLE
EQ3 does not appear on MQTT resp. iobroker

### DISCUSSION THREAD

**Discussion by dog-rgb** [arendst/Tasmota#13971] [Q&A] [answered]
Hi,

I am using v10.1.0.1 on ESP32 with tasmota32-bluetooth.bin

There is currently one EQ3 device in range for the ESP32 and this is also found according to the console:
```
14:59:01.805 EQ3 poll cycle starting
14:59:01.807 EQ3 001A2211D4EF: Op queued len now 1
14:59:01.809 EQ3 001A2211D4EF:Op dequeued len now 0
14:59:03.842 MQT: stat/EQ3/001A2211D4EF = {"cmd":"poll","result":"ok","MAC":"001A2211D4EF","tas":"tasmota-5DE928-2344","RSSI":-78,"stattime":1639058343,"temp":18.0,"posn":6,"mode":"manual","hassmode":"heat","boost":"inactive","dst":"set","window":"closed","state":"unlocked","battery":"GOOD"}
14:59:03.879 MQT: tele/tasmota_5DE928/BLE = {"BLEOperation":{"opid":"0","stat":"7","state":"DONENOTIFIED","MAC":"001A2211D4EF","u":"128","svc":"3e135142-654f-9090-134a-a6ff5bb77046","char":"3fa4585a-ce4a-3bad-db4b-b8df8179ea09","notifychar":"d0e8434d-cd29-0996-af41-6c90f4e0eb2a","write":"03150C090E3B01000000000000000000","notify":"020109060424"}}
```

But the EQ3 is not visible in iobroker, is there something I missed?

MQTT is configured correctly, there are some temp sensors in range which gets reported to iobroker. But not the EQ3.

Any other information that helps/is needed?

**Comment by barbudor** [comment, marked as answer]
Have you read https://tasmota.github.io/docs/EQ3-TRV/ ?
There are commands dedicated to TRV

**Reply by dog-rgb** [reply]
Yes, I have seen them.
But I somehow expected at least the status report as separate data points automatically transferred to iobroker like it is done for the temperature sensors for example. The status report is gathered between tasmota and the EQ3 devices as visible in the console output.
If this is not the case, it is not clear to me how to transfer data between iobroker and tasmota for the EQ3 devices.

**Reply by dog-rgb** [reply]
Ok, it seems I got a step further. I used the sonoff adapter as I do with other tasmota devices. Now I changed to MQTT adapter and as a result the expected data points appear.

Now I need to learn to extract the data from the JSON values.
